src/core/UniswapV4.sol

```diff
@@ -298,7 +297,7 @@ abstract contract UniswapV4 is SettlerAbstract {
             }
             // TODO: price limits
             params.sqrtPriceLimitX96 = uint160(
-                zeroForOne.ternary(uint160(4295128740), uint160(1461446703485210103287273052203988822378723970341))
+                (!zeroForOne).ternary(uint160(1461446703485210103287273052203988822378723970341), uint160(4295128740))
```

Comment:
is this an improvement? 😅  How so?

Reply:
Yeah, it's actually quite a big improvement. It's due to how `ternary` is implemented and `solc`'s ability to optimize constant expressions. Basically, the opcodes emitted are the same, but the constants involved are shorter